### ISSUE TITLE
deCONZ stopped working after addon update

### ISSUE THREAD

**Issue by unlucio** [home-assistant/addons#2321]
### Describe the issue you are experiencing

Just updated deCONZ to the latest version and the integration stopped working and now I cannot access the Phoscon interface.

### What type of installation are you running?

Home Assistant Supervised

### Which operating system are you running on?

Other (e.g., Raspbian/Raspberry Pi OS/Fedora)

### Which add-on are you reporting an issue with?

deCONZ

### What is the version of the add-on?

6.11.0

### Steps to reproduce the issue

1. updated the addon to the latest version
2. HA integration started showing as not working in the `config/integrations` panel
3. removed HA integration
4. tried to re-add it
5. not integration asks for a "host" before installing (I've no idea what is the host as it's one of the internal add-ons), searched the doc and the web without success
<img width="405" alt="Screenshot 2021-12-20 at 18 54 23" src="https://user-images.githubusercontent.com/778799/146811402-eb223aa5-8938-40d7-b6f6-a8fc45aa051d.png">
6. went in the deCONZ panel
7. selected "Phoscon Web App"
8. tries to load
9. gets stuck at "Please wait until the search for gateways has finished."
<img width="684" alt="Screenshot 2021-12-20 at 18 48 38" src="https://user-images.githubusercontent.com/778799/146811254-31e61fd2-be2f-49aa-be0b-5b7da312940f.png">
10. double-checked I'm accessing the interface via IP and not via host name, since it's a know deCONZ issue: yes I am
11. started banging my head on the table :(

### Anything in the Supervisor logs that might be useful for us?

```txt
Nothing in the logs about deCONZ
```

### Anything in the add-on logs that might be useful for us?

```txt
nothing relevant, just http access logs
```

### Additional information

Home Assistant 2021.12.3

**Comment by ShlomiPorush** [comment]
update to version 6.11.1

**Comment by frenck** [comment]
This issue has been fixed, as mentioned above by Shlomi, in 6.11.1, please update.

**Comment by unlucio** [comment]
oh thank you, I guess I had a lucky update in the narrow windows between ..0 and ..1 😶
